sotodlib/preprocess/processes.py

```diff
@@ -1760,6 +1760,34 @@ def process(self, aman, proc_aman):
         if self.process_cfgs:
             pointing_model.apply_pointing_model(aman)
 
+class CorrectIIRParams(_Preprocess):
+    """Correct missing iir_params by default values if observation is
+    within a given time range.
+
+    Example config block::
+
+        - name: "correct_iir_params"
+          time_range: ["2023-01-01", "2025-01-01"]
+          process: True
+
+    .. autofunction:: sotodlib.obs_ops.utils.correct_iir_params
+    """
+    name = "correct_iir_params"
+    def __init__(self, step_cfgs):
+        self.time_range = step_cfgs.get('time_range')
+        super().__init__(step_cfgs)
+
+    def process(self, aman, proc_aman):
+        import datetime
+        from sotodlib.obs_ops import correct_iir_params
+        if self.time_range is not None:
```

Comment:
It is odd to do nothing, if that parameter is missing.  It should either be:
- an error to omit time_range argument, or
- time_range argument is optional.